### ISSUE TITLE
Should aim run in docker？

### ISSUE THREAD

**Issue by LyazS** [aimhubio/aim#275]
I install docker in Ubuntu, then I start it and my docker service is active and add the my user in docker group also. 

Then I run `aim up
` in my anaconda environment. But it says Oops I don’t have docker installed. 

Could u please specify how to run the` aim up `?

**Comment by gorarakelyan** [comment]
Hi @LyazS,

Try to run `docker version` command from your environment to check if docker was properly installed.

Also please make sure to close and reopen shell for docker configurations to take effect.

**Comment by LyazS** [comment]
Yes I have done the procedures. Even I have successfully run the docker hello world example. But, `aim up` doesn’t work

**Comment by gorarakelyan** [comment]
Thanks for the response,

Please try to run the following python code snippet from your training environment where `aim` and `docker` are installed.
It will check if python docker SDK is installed(comes along with aim) and docker service is accessible.

```py
import docker
client = docker.from_env()
client.ping()
```

**Comment by SGevorg** [comment]
Hi @LyazS, does still `aim up` not work for you? Did none of these steps help?

**Comment by LyazS** [comment]
I install `docker-py` and successfully import docker. 
But when I run `clinet.ping()`, It says:
`requests.exceptions.ConnectionError: ('Connection aborted.', PermissionError(13, 'Permission denied'))`

Then, `aim up` still not work

**Comment by SGevorg** [comment]
@LyazS that's so bizarre! 
Please join Aim community slack [here](https://join.slack.com/t/aimstack/shared_invite/zt-ik4s4zb7-jGg4WIMu4s3NL3YAkOHVXA) and we are happy to do a quick screenshare session to help you resolve this issue and get up and running soon. 

We are also preparing a cheat list + FAQ for docker. I will report back the cause and the results in here once we solve the issue.

**Comment by gorarakelyan** [comment]
@LyazS, it seems something is wrong with docker permissions.
I see python docker client works correctly only when you start python shell with superuser privileges.

The following must help:
 - Add your user to docker user group `sudo usermod -aG docker $USER`
 - Restart docker service `sudo systemctl restart docker`
 - Restart shell

If it does not help, please join [Aim slack workspace](https://join.slack.com/t/aimstack/shared_invite/zt-ik4s4zb7-jGg4WIMu4s3NL3YAkOHVXA) so we could help you to resolve this issue quickly.
